components/proxy/src/main/java/com/hotels/styx/routing/handlers/RouteRefLookup.java

```diff
@@ -30,7 +30,10 @@
  * Resolves a routing object reference from route database.
  */
 public interface RouteRefLookup {
-    HttpHandler apply(RoutingObjectReference route);
+    // Consider modifying this interface to return Optional<RoutingObject>.
```

Comment:
Is this comment still relevant?

Reply:
Yeah. It was originally a TODO, but then decided to leave it as a normal comment. I'm bit indecisive at the moment.